tests/tasks/test_workflows.py

```diff
@@ -321,25 +321,29 @@ def test_MIP(
     parallelization_list = copy_ome_zarr_hcs_plate(
         zarr_urls=zarr_urls,
         zarr_dir=str(zarr_path),
-        overwrite=True,
+        overwrite_images=True,
+        re_initialize_plate=True,
     )["parallelization_list"]
     debug(parallelization_list)
 
     # Run again, with overwrite=True
     parallelization_list_2 = copy_ome_zarr_hcs_plate(
         zarr_urls=zarr_urls,
         zarr_dir=str(zarr_path),
-        overwrite=True,
+        overwrite_images=True,
+        re_initialize_plate=True,
     )["parallelization_list"]
     assert parallelization_list_2 == parallelization_list
 
     # Run again, with overwrite=False
-    with pytest.raises(OverwriteNotAllowedError):
-        _ = copy_ome_zarr_hcs_plate(
-            zarr_urls=zarr_urls,
-            zarr_dir=str(zarr_path),
-            overwrite=False,
-        )
+    # TODO check with joel if this should be removed
```

Comment:
Given that we test the overwrite behavior in other tests in test_unit_copy_hcs.py, fine to remove it here